> Can you provide more detail in the PR desc, eg what problem was observed?

It was on Mike's other Shipyard PR (https://github.com/submariner-io/shipyard/pull/585):

```
[linting]$ make gitlint
gitlint --commits origin/devel..HEAD
Traceback (most recent call last):
  File "/usr/bin/gitlint", line 5, in <module>
    from gitlint.cli import cli
  File "/usr/lib/python3.9/site-packages/gitlint/cli.py", line 14, in <module>
    from gitlint.git import GitContext, GitContextError, git_version
  File "/usr/lib/python3.9/site-packages/gitlint/git.py", line 4, in <module>
    import arrow
  File "/usr/lib/python3.9/site-packages/arrow/__init__.py", line 2, in <module>
    from .api import get, now, utcnow
  File "/usr/lib/python3.9/site-packages/arrow/api.py", line 12, in <module>
    from arrow.arrow import TZ_EXPR, Arrow
  File "/usr/lib/python3.9/site-packages/arrow/arrow.py", line 31, in <module>
    from dateutil import tz as dateutil_tz
  File "/usr/lib/python3.9/site-packages/dateutil/tz/__init__.py", line 2, in <module>
    from .tz import *
  File "/usr/lib/python3.9/site-packages/dateutil/tz/tz.py", line 19, in <module>
    import six
ModuleNotFoundError: No module named 'six'
make: *** [Makefile.inc:119: gitlint] Error 1
```

--- a/package/Dockerfile.shipyard-linting
+++ b/package/Dockerfile.shipyard-linting
@@ -17,11 +17,12 @@ ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH} \
 # nodejs           | Used by markdownlint
 # npm              | Installing markdownlint (Removed afterwards)
 # py3-pip          | Installing gitlint (Removed afterwards)
+# py3-six          | Required by gitlint
 # shellcheck       | Shell script linting
 # upx              | Compressing executables to get a smaller image
 # yamllint         | YAML linting
 
-RUN apk add --no-cache bash findutils git grep make nodejs shellcheck upx yamllint && \
+RUN apk add --no-cache bash findutils git grep make nodejs py3-six shellcheck upx yamllint && \
     apk add --no-cache --virtual installers npm py3-pip && \
     npm install -g markdownlint-cli && \
     pip install gitlint && \

